website/content/v1.2/talos-guides/install/single-board-computers/nanopi_r4s.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Friendlyelec Nano PI R4S"
+description: "Installing Talos on a Nano PI R4S SBC using raw disk image."
+aliases:
+  - ../../../single-board-computers/nanopi_r4s
+---
+
+## Prerequisites
+
+You will need
+
+- `talosctl`
+- an SD card
+
+Download the latest `talosctl`.
+
+```bash
+curl -Lo /usr/local/bin/talosctl https://github.com/siderolabs/talos/releases/download/{{< release >}}/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
+chmod +x /usr/local/bin/talosctl
+```
+
+## Download the Image
+
+Download the image and decompress it:
+
+```bash
+curl -LO https://github.com/siderolabs/talos/releases/download/{{< release >}}/metal-rockpi_4-arm64.img.xz
```

Comment:
this still mentions "rockpi_4", is that intended?

Reply:
Good thing you caught the typo, no that's wrong and I'll fix and rebase the PR then. The generated image is being called metal-nanopi_r4s-arm64.img.xz of course.